### PR TITLE
Remove Poet-js & add WorkAttributes Interface

### LINKED ISSUE #2: Method getApiTokens return type is wrong
We have to update the return type of the method getApiTokens

https://github.com/poetapp/frost-client/blob/master/src/Frost.ts#L291

Now the typedef return is the following

```
  getApiTokens(token: string): Promise<ReadonlyArray<{
        apiToken: string;
        dateCreated: string;
    }>>

```
but isn't correct, now the API return only the tokens. Therefore, we have to update the typedef return for:

```
 getApiTokens(token: string): Promise<{ apiTokens: ReadonlyArray<string> }>
```

Also, we have to update the Frost web where use this method

https://github.com/poetapp/frost-web/blob/master/src/sagas/GetApiTokens.ts#L8

### PR DESCRIPTION
**Pull request by jwicks31**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

### Release

Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 

We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).

## Description of Changes

`Fixes #1, Resolves #2`

The poet-js dependency was causing issues with Frost-web and Netlify. It was only being used to add this interface, so I removed the dependency and added the interface.

### CODE REVIEW

File: src/Frost.ts [hunk 1]
```diff
@@ -9,6 +8,19 @@ export interface Configuration {
   readonly timeout?: number
 }
 
+export interface ClaimAttributes {
```
Comment (krobi64) [inline src/Frost.ts]: Both of these interfaces are going to change once `frost-api` and `node` are updated to `poet-js` 5.x

If these were the only things being used, what issues were arising?

Comment (jwicks31) [reply src/Frost.ts]: Whenever I am trying to update [Frost-web](https://github.com/poetapp/frost-web/pull/347) to the latest version of `frost-client` it is causing issues with Netlify. @wzalazar and I tried to debug and were able to get Travis passing with some `before_install` commands. But, we didn't have any luck with Netlify.

**Comment by jwicks31** [comment]
@krobi64 I'm not exactly sure how poet-js is causing the issue, but whenever I tried updating Frost-web with the latest version of frost-client t wasn't able to build in Netlify. 

We can confirm that it is poet-js that is causing the issue because I have installed this branch inside of frost-web and the Netlify build is now working. 

The only changes to this branch are removing poet-js and adding the interface. I had to remove the dist folder from git-ignore to be able to install this branch directly inside of my Frost-web branch [here](https://github.com/poetapp/frost-web/pull/347)

File: src/Frost.ts (hunk 1 above)
Comment (krobi64) [reply src/Frost.ts]: Ok, I think this isn't an issue. Very shortly both of these interfaces are going away.

Comment (jwicks31) [reply src/Frost.ts]: Whenever they do change will they need to be imported into frost-client from poet-js? If so I think we will have the same issue at that point.

**Pull request merged**